pkg/util/net/http.go

```diff
@@ -70,6 +73,19 @@ func SetTransportDefaults(t *http.Transport) *http.Transport {
 	return t
 }
 
+// SetTransportDefaults applies the defaults from http.DefaultTransport
+// for the Proxy, Dial, and TLSHandshakeTimeout fields if unset
+func SetTransportDefaults(t *http.Transport) *http.Transport {
+	t = SetOldTransportDefaults(t)
+	// Allow HTTP2 clients but default off for now
+	if s := os.Getenv("ENABLE_HTTP2"); len(s) > 0 {
```

Comment:
/cc @jeremyeder @mffiedler FYI.  New option to reduce connection counts, defaulted off.